eternalcore-core/src/main/java/com/eternalcode/core/feature/home/placeholder/HomeLimitGlobalPlaceholderImpl.java

```diff
@@ -0,0 +1,24 @@
+package com.eternalcode.core.feature.home.placeholder;
+
+import com.eternalcode.core.configuration.implementation.PluginConfiguration;
+import com.eternalcode.core.placeholder.PlaceholderReplacer;
+import org.bukkit.entity.Player;
+
+public class HomeLimitGlobalPlaceholderImpl implements PlaceholderReplacer {
```

Comment:
The logic of this placeholder makes no sense to me, it returns the home limit for the highest rank in the config. I think it should be deleted

Reply:
![gif](https://media.tenor.com/tgcRIPI6pmAAAAAC/mr-bean-mr-bean-holiday.gif)